src/corelibs/U2Lang/src/support/WorkflowDebugMessageParser.cpp

```diff
@@ -60,6 +61,7 @@ void WorkflowDebugMessageParser::setSourceData(const QQueue<Message> &initSource
         QVariantMap data = message.getData().toMap();
         foreach (const QString &key, data.keys()) {
             if (key.contains(MESSAGE_PATH_DELIMETER)) {
+                coreLog.trace(QObject::tr("WorkflowDebugMessageParser.cpp:64 'key' = %1").arg(key));
```

Comment:
nit: 
1) tr is not needed in trace() like this
2) use `__LINE__` when you need line number. Or remove the line number: the message is already unique for devs.